addons/mail/static/src/discuss/core/common/action_panel.scss

```diff
@@ -1,4 +1,4 @@
-.o-mail-ActionPanel:not(.o-mail-ActionPanel-chatter) {
+.o-mail-ActionPanel:not(.o-mail-ActionPanel-exclude) {
```

Comment:
With simpler classname as modifier, this is `.o-mail-ActionPanel:not(.o-inChatWindow)` (or `:not(.o-fullscreen)`, whichever you pick)